services/src/kilda-core/kilda-persistence-api/src/main/java/org/openkilda/persistence/repositories/FlowRepository.java

```diff
@@ -43,4 +43,6 @@
     void delete(FlowPair flowPair);
 
     void updateStatus(String flowId, FlowStatus status);
+
+    Collection<FlowPair> findAllFlowPairsForIsl(SwitchId srcSwitchId, int srcPort, SwitchId dstSwitchId, int dstPort);
```

Comment:
I'd rename the method to findAllFlowPairsWithSegment as you don't work with ISL in this method.

Reply:
Renamed.